test/abstract_syntax_tree_location_data.coffee

```diff
@@ -407,3 +407,39 @@ test "AST location data as expected for Index node", ->
       end:
         line: 1
         column: 8
+
+test "AST location data as expected for Parens node", ->
+  testAstLocationData '(hmmmmm)',
+    type: 'Identifier'
+    start: 1
+    end: 7
+    range: [1, 7]
+    loc:
+      start:
+        line: 1
+        column: 1
+      end:
+        line: 1
+        column: 7
```

Comment:
Are you checking these numbers against the location data reported by astexplorer.net or some other source? Also, I don’t see `range` there or as part of the Babel spec; why are we including it?

I wonder if perhaps we can get this location data from Babel and compare against ours. Would that only work for code like `(hmmmmm)` that’s identical between CoffeeScript and JavaScript? If it works for all code, we could put the numbers comparison in the `testAstLocationData` helper and none of the tests in this file would need this second argument (the `expected` object with the location data).